api/api_helpers_test.go

```diff
@@ -82,3 +87,34 @@ func getAPIConfig() *API {
 		Servers: servers,
 	}
 }
+
+// setupTestContainer initializes and starts the PostgreSQL test container.
+func setupPostgreSQLTestContainer(ctx context.Context, t *testing.T) (string, nat.Port) {
```

Comment:
Shouldn't we make this reusable and not copy it across packages? Any thoughts?

Reply:
We can’t import code from a test package or an XXX_test.go file directly. But I agree, it makes sense to create a package for reusable code. Normally, for internal functions, the files should go in an internal folder (like in the Go docs: https://golang.org/doc/go1.4#internalpackages). Since we don’t really follow the pkg (public) and internal (private) structure, we could just create a folder in the root.

Maybe something like `testhelpers/container_helpers.go` or `internal/testhelpers/container_helpers.go`?

@Mostafa, what do you think?

Reply:
`testhelper/*` sounds good. We can refactor later to follow public/internal.